packages/manager/apps/hub/webpack.config.js

```diff
@@ -1,32 +1,27 @@
-const _ = require('lodash');
 const glob = require('glob');
 const path = require('path');
 const webpack = require('webpack'); // eslint-disable-line
 const merge = require('webpack-merge');
 const webpackConfig = require('@ovh-ux/manager-webpack-config');
 
 module.exports = (env = {}) => {
-  const REGION = _.upperCase(env.region || process.env.REGION || 'EU');
-
   const { config } = webpackConfig(
     {
       template: './src/index.html',
       basePath: './src',
       lessPath: ['./node_modules'],
       root: path.resolve(__dirname, './src'),
     },
-    REGION ? Object.assign(env, { region: REGION }) : env,
+    env,
```

Comment:
just removed REGION related parts.

Reply:
Not sure, as `region` is expected in `env` in `@ovh-ux/manager-webpack-dev-server`

Reply:
Fixed by using `process.env.region` in `@ovh-ux/manager-webpack-dev-server` config